prac_05/state_names.py

```diff
@@ -0,0 +1,21 @@
+"""
+CP1404/CP5632 Practical
+State names in a dictionary
+File needs reformatting
+"""
+
+CODE_TO_NAME = {"QLD": "Queensland", "NSW": "New South Wales", "NT": "Northern Territory", "WA": "Western Australia",
+                "ACT": "Australian Capital Territory", "VIC": "Victoria", "TAS": "Tasmania"}
+# print(CODE_TO_NAME)
+
+for state_code, state_name in CODE_TO_NAME.items():
+    print(f"{state_code:3} is {state_name}")
+
+state_code = input("Enter short state: ").upper()
+while state_code != "":
+    try:
+        state_name = CODE_TO_NAME[state_code]
+        print(state_code, "is", CODE_TO_NAME[state_code])
+    except KeyError:
+        print("Invalid short state")
+    state_code = input("Enter short state: ").upper()
```

Comment:
There's another requirement for this one. 
# Write a loop that prints all the states and names neatly lined up with string formatting.
Don't for get to implement variables for the maximum lengths of the code and the name.